src/rgw/rgw_rest_swift.cc

```diff
@@ -572,7 +572,10 @@ int RGWCopyObj_ObjStore_SWIFT::get_params()
   dest_bucket_name = s->bucket_name_str;
   dest_object = s->object.name;
 
-  if (!s->info.x_meta_map.empty()) {
```

Comment:
I'm pretty sure the _ATTRSMOD_MERGE_ mode should be applied for the _generic_attrs_ as well. Please take a look: https://github.com/rzarzynski/ceph/blob/wip-10645/src/rgw/rgw_op.cc#L2351.